stacker/hooks/aws_lambda.py

```diff
@@ -100,7 +153,20 @@ def _head_object(s3_conn, bucket, key):
 
 
 def _ensure_bucket(s3_conn, bucket):
-    """Create an S3 bucket if it doesn't already exist"""
+    """Create an S3 bucket if it does not already exist.
+
+    Args:
+        s3_conn (.S3.Client): S3 connection to use for operations.
```

Comment:
I was actually experimenting with cross-referencing to the boto docs, which seem to be quite broken - they don't prefix any of the classes with the correct package in the indexes.

I think the correct type is actually `botocore.client.S3`.